index.html

```diff
@@ -5,8 +5,168 @@
+                <h3 class="feature d-flex flex-column justify-content-between align-items-center">Main Program <span></span></h3>
+                <ul class="feature-box">
+                    <li>
+                        <img src="img/program_icon_01.png" alt="Lecture Icon" class="program_icon">
+                        <h3 class="program_title">Lecture</h3>
+                        <p class="program_descp">Listen to the speakers from various countries about the messages of sharing and opening.</p>
+                    </li>
+                    <li>
+                        <img src="img/program_icon_02.png" alt="Lecture Icon" class="program_icon">
+                        <h3 class="program_title">CC Exhibition</h3>
+                        <p class="program_descp">Appreciate various creations applying CC license of artists, organized from National Musuem of Malawi</p>
+                    </li>
+                    <li>
+                        <img src="img/program_icon_03.png" alt="Lecture Icon" class="program_icon">
+                        <h3 class="program_title">Forum</h3>
+                        <p class="program_descp">Have the time to share your thoughts and opinions with experts for each topic.</p>
+                    </li>
+                    <li>
+                        <img src="img/program_icon_04.png" alt="Lecture Icon" class="program_icon">
+                        <h3 class="program_title">Workshop</h3>
+                        <p class="program_descp">Try creating your own work using open source license rather than just watching at it.</p>
+                    </li>
+                    <li>
+                        <img src="img/program_icon_05.png" alt="Lecture Icon" class="program_icon">
+                        <h3 class="program_title">CC Ignite</h3>
+                        <p class="program_descp">Get opportunities to network with CC affiliates around the world, also after the summit.</p>
+                    </li>
+                </ul>
+                <button class="feature mobile">Join CC Global Summit 2015</button>
+                <a href="about.html" class="feature_link desktop">See the whole program</a>
+            </div>
+        </section>
+        <!-- Feature Section Ends Here-->
+
+        <!-- Speakers Section Starts Here -->
+
+        <section class="speakers d-flex flex-column align-items-center justify-content-between">
+            <h3 class="feature d-flex flex-column justify-content-between align-items-center" id="speakers_title">Featured Speakers<span></span></h3>
+            <ul class="d-flex flex-sm-column flex-md-row justify-content-between align-items-center" id="speakers_container">
+                <!-- Speakers details will be loaded by javascript -->
+            </ul>
+            <button class="more_speakers mobile">More<img src="img/down-arrow.png" class="down_icon" alt="Down"></button>
+        </section>
```

Comment:
- [ ] To make your project more responsive, Kindly display your Speakers in a column in the mobile version as on the original design 👍 
- [ ] Also, Please, make sure your `button` look the same on the design 😃 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-05-24 140121](https://github.com/hmunish/capstone-htmlcssjs/assets/57408419/e18c7cd9-e5a7-45c0-9221-2b01ffb19166)| |![Screenshot 2023-05-24 135517](https://github.com/hmunish/capstone-htmlcssjs/assets/57408419/8149345c-6542-41cb-9620-20e6a868f921)| |